src/core/storage/storagekeys.js

```diff
@@ -40,3 +42,4 @@ export default {
   QUERY_SOURCE: 'query-source',
   HISTORY_POP_STATE: 'history-pop-state'
 };
+export default StorageKeys;
```

Comment:
this was so my intellisense could autoimport StorageKeys correctly,
autoimport without this
`import storagekeys from '../storagekeys'`
with this
`import StorageKeys from '../storagekeys'`
not sure if other people have this issue